docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_label: Artifactory
+---
+
+# Artifactory Webhook Receiver
+
+The Artifactory Webhook Receiver responds to `pushed` events originating from 
+Artifactory repositories by _refreshing_ all `Warehouse` resources subscribed to 
+those repositories.
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+An Artifactory webhook receiver must reference a Kubernetes `Secret` resource 
+with a `secret-token` key in its data map. This
+[shared secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by
+Artifactory to sign requests any by the receiver to verify those signatures.
+
+:::note
+The following commands are suggested for generating and base64-encoding a
+complex secret:
+
+```shell
+secret_token=$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)
+echo "Secret token: $secret_token"
+echo "Encoded secret token: $(echo -n $secret_token | base64)"
+```
+
+:::
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+  name: kargo-demo
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: artifactory-wh-secret
+  namespace: kargo-demo
+  labels:
+    kargo.akuity.io/cred-type: image
+data:
+  repoURL: <jfrog-instance>.jfrog.io/<repo-key>/<image-name>
+  username: <jfrog username>
+  password: <jfrog docker login password>
```

Comment:
What are these being used for? Looking at #4498 I couldn't quite figure that out.

Reply:
For authenticating with a private repo. I added a note for this [here](https://github.com/akuity/kargo/pull/4514/files#diff-1e57838d38a8e829eb2e95a50b1ac7a0e8541eb5a9ad6ed8def0d2f1da8db184R69).

Reply:
Oh... nevermind. I see what's going on. Will ping you offline.